syntax = "proto3";

package PbApp;

message UserData
{
    string username = 1;
    string token = 2;
    string refreshToken = 3;
    double expiry = 4;
    uint64 role = 5;
}